.github/workflows/ci.yaml

```diff
@@ -0,0 +1,37 @@
+name: Build and Test
+
+on:
+  push:
+
+env:
+  CONTAINER_IMAGE: pdffonts-test:${{ github.sha }}-${{ github.run_id }} 
+  CONTAINER_NAME: pdffonts_test_run_${{ github.sha }}-${{ github.run_id }}
+
+jobs:
+  ci:
+    name: Build & Test
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Build container image
+        run: docker build --build-arg NODE_ENV=development -t "$CONTAINER_IMAGE" .
```

Comment:
this doesn't need a test env?

Reply:
Nope just needs a debug build of the native module 🎉